src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+// store.js
+
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+export default configureStore({
+  reducer: {
+    books: booksReducer,
+  },
+});
```

Comment:
- [ ] Kindly ensure to use `combineReducers` to create your reducer, so as to combine both reducers before creating a store.